dftimewolf/lib/processors/workspace_audit_timesketch.py

```diff
@@ -108,14 +126,14 @@ def _AddMessageString(self, timesketch_record: Dict[str, Any]) -> None:
           actor = (timesketch_record.get('actor_email') or
                    timesketch_record.get('actor_profileId') or
                    timesketch_record.get('actor_key'))
-          message += '"{0:s}"'.format(actor)
+          message += '{0:s}'.format(actor)
```

Comment:
What about 

```suggestion
          message += actor
```

Reply:
I can't guarantee that 'actor' is a string here - it could be a number (id) or something else.

Reply:
Then we want something like `str(actor)` or '{0!s}.format(actor)', no?

Reply:
Good point - changed